examples/catalog/test/expansion_tile_sample_test.dart

```diff
@@ -0,0 +1,80 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import '../lib/expansion_tile_sample.dart' as expansion_tile_sample;
+import '../lib/expansion_tile_sample.dart' show Entry;
+
+void main() {
```

Comment:
That's a shame.  By verifying that the app is doing what's expected, I'm essentially testing the daylights out of ExpansionTile.